src/Altinn.Platform/Altinn.Platform.PDF/src/main/java/altinn/platform/pdf/models/FormLayoutElement.java

```diff
@@ -47,5 +48,9 @@ public String getType() {
 
   public void setMaxCount(int maxCount) { this.maxCount = maxCount; }
 
+  public List<String> getDataTypeIds() { return dataTypeIds; }
```

Comment:
An option to this boilerplatecode is to use Lombok. Just add "@Getter" and "@Setter" to the class. The library is already included as a dependency.

Reply:
Omg, that's so much better ❤️ 😄